resources/msg.go

```diff
@@ -114,7 +123,6 @@ func (obj *MsgRes) journalPriority() journal.Priority {
 	case "Debug":
 		return journal.PriDebug
 	}
-	return journal.PriNotice
```

Comment:
i made the assumption that it was there because you didn't have validation and wanted it to default to Notice when a match was not found.